exporter/sapmexporter/go.mod

```diff
@@ -4,10 +4,11 @@ go 1.12
 
 require (
 	github.com/open-telemetry/opentelemetry-collector v0.2.1-0.20191119152140-567e1046cefa
-	github.com/signalfx/sapm-proto v0.0.3-0.20191211161027-5e20f16d64d6
+	github.com/signalfx/sapm-proto v0.1.0
 	github.com/stretchr/testify v1.4.0
 	go.uber.org/atomic v1.5.1 // indirect
 	go.uber.org/multierr v1.4.0 // indirect
 	go.uber.org/zap v1.13.0
+	google.golang.org/genproto v0.0.0-20191115221424-83cc0476cb11 // indirect
```

Comment:
go mod tidy added this. Go is seeing this as a dependency of sapm-proto. This is not a big deal but I'll look into it later. For sapm-proto, this should really be a build dependency and not pulled in here.